src/components/icon/icon.css

```diff
@@ -0,0 +1,85 @@
+@import "../../../src/styles/colors.css";
+
+:root {
```

Comment:
I just realized that this is a great pattern:

Others can import this stylesheet for both variables and classes. 

This is way different than the SASS counterpart where we had to split the variables, the mixins, etc, into different files.